File: AngluarPI-main/src/app/model/event.ts
import { Ticket } from "./ticket";

export enum Statuss {
    Confirmed = 'Confirmed',
    On_Hold = 'On_Hold',
    Canceled = 'Canceled',
    Completed = 'Completed'
  }

export class Event {
        idEvent!: number;
        description!: string;
        date!: Date;
        number_of_tickets!: number;
        additional_notes!: string;
        place!: string;
        statuss!: Statuss;
        tickets!: Ticket[]
}

File: SpringBoot-Pi-main/src/main/java/com/example/pokerplanninpi/entity/Event.java
package com.example.pokerplanninpi.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDate;

@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Event {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long idEvent;
    private String description;
    private LocalDate date;
    private int Number_of_tickets;
    private String additional_notes;
    private String place;

    @Enumerated(EnumType.STRING)
    private Statuss statuss;






}
